my-react-app/src/redux/store.js

```diff
@@ -1,10 +1,10 @@
 import { createStore, combineReducers } from 'redux';
 import { reducer as formReducer } from 'redux-form';
 
-import Reducer from './game/reducer';
+import { reducer as gameReducer } from './game/reducer';
 
 const reducers = {
-  Reducer,
+  gameReducer,
```

Comment:
Now that you have 2 reducers, the state has 2 branches. You have to specify the branch when you map the state to props.
```
const mapStateToProps = state => ({
  history: state.history,
  stepNumber: state.stepNumber,
  xIsNext: state.xIsNext
});
```

For example, instead of `history: state.history,` it's `history: state.game.history,`